### PR TITLE
Feature/54082

### PR DESCRIPTION
**Pull request by bartdominiak**
#144

### CODE REVIEW

File: Snowdog_SolrNavigation/templates/grid.phtml [hunk 1]
```diff
@@ -106,40 +113,40 @@ $solrDataHelper = $this->helper('Snowdog\SolrNavigation\Helper\Data');
...
-                                class="catalog-grid-item__secondary-action"
-                                action="<?= $block->escapeHtml($block->getAddToUrl('wishlist', $_product['entity_id'])) ?>"
-                                method="post"
-                                data-mage-init='{ "validation": {} }'
-                            >
-                                <button
-                                    class="button button--icon"
-                                    type="submit"
+                            <div class="catalog-grid-item__secondary-action">
+                                <a
```
Comment (Aekal) [inline Snowdog_SolrNavigation/templates/grid.phtml]: Could you change it to `<button type="button">` without empty `href`?

File: Snowdog_SolrNavigation/templates/grid.phtml [hunk 2]
```diff
@@ -106,40 +113,40 @@ $solrDataHelper = $this->helper('Snowdog\SolrNavigation\Helper\Data');
...
-                                method="post"
-                                data-mage-init='{ "validation": {} }'
-                            >
-                                <button
-                                    class="button button--icon"
-                                    type="submit"
+                                </a>
+                            </div>
+                            <div class="catalog-grid-item__secondary-action">
+                                <a
```
Comment (Aekal) [inline Snowdog_SolrNavigation/templates/grid.phtml]: `<button>`

File: Snowdog_SolrNavigation/templates/grid.phtml [hunk 3]
```diff
@@ -106,40 +113,40 @@ $solrDataHelper = $this->helper('Snowdog\SolrNavigation\Helper\Data');
...
-                                action="<?= $block->escapeHtml($block->getAddToUrl('wishlist', $_product['entity_id'])) ?>"
-                                method="post"
-                                data-mage-init='{ "validation": {} }'
-                            >
-                                <button
-                                    class="button button--icon"
-                                    type="submit"
+                            <div class="catalog-grid-item__secondary-action">
+                                <a
+                                    class="button button--icon action towishlist"
```
Comment (Aekal) [inline Snowdog_SolrNavigation/templates/grid.phtml]: Do we need `action towishlist` classes?

Comment (Aekal) [reply Snowdog_SolrNavigation/templates/grid.phtml]: Same for `action tocompare` + in `list.phtml`

Comment (bartdominiak) [reply Snowdog_SolrNavigation/templates/grid.phtml]: Unecessary. Deleted.

**Pull request merged**